dinhthuankhang/13_08/Bài 2 DrawTriangle.kts

```diff
@@ -0,0 +1,24 @@
+var side = ""
+val point = "*"
+val space = " "
+val numberSpaceOut = arrayOf(2,4,6,8,10,12,14,16,18,20,22,24,26,28,30,32,34,36,38,40,42,44,46,48,50)
+val numberSpaceIn = arrayOf(1,3,5,7,9,11,13,15,17,19,21,23,25,27,29,31,33,35,37,39,41,43,45,47,49)
+var numIn = 1
+print("Write side length: ")
+var length = readLine()!!.toInt()
+if (length >= 3) {
+    length -= 2
+    var numSpaceOut = numberSpaceOut[length]
```

Comment:
cũng là 1 cách khá sáng tạo, nhưng a nghĩ mình có thể có 1 giải pháp tối ưu hơn ở đây. Nếu người dùng muốn in ra tam giác có cạnh bằng 100 chẳng hạn, thì chương trình này sẽ bị lỗi mất :sweat_smile:

Reply:
vâng ạ :)))
thế nên em có hỏi có cách nào khai báo mảng full số chẵn hay số lẻ không ạ :))))